fuzzing/extract_text_fuzzer.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import atheris
+
+from fuzz_helpers import EnhancedFuzzedDataProvider
+
+with atheris.instrument_imports():
+    from pdf_utils import PDFValidator, prepare_pdfminer_fuzzing
+    from pdfminer.high_level import extract_text
+
+from pdfminer.psparser import PSException
+
+
+def TestOneInput(data: bytes):
```

Comment:
The rest of pdfminer is type checked with mypy. That would error on the missing return type here. What do you think of adding the `fuzzing` directory to the `noxfile.py`? 

That would also enable black formatting and linting.